infra/cpr/plumbing/main.py

```diff
@@ -25,10 +25,62 @@ class Plumbing:
     service_role: aws.iam.Role
 
     def __init__(self):
+        self.default_vpc = aws.ec2.DefaultVpc(
+            "default-vpc",
+            tags={
+                "Name": "Default VPC",
+            },
+        )
+
+        # Security groups control traffic flow between applications running inside our VPC
+        security_group_bastion = aws.ec2.SecurityGroup(
+            "bastion-security-group",
+            description="Enable SSH access",
+            vpc_id=self.default_vpc.id,
+            ingress=[
+                aws.ec2.SecurityGroupIngressArgs(
+                    protocol="tcp",
+                    from_port=22,
+                    to_port=22,
+                    cidr_blocks=["0.0.0.0/0"],
+                )
+            ],
+            egress=[
+                aws.ec2.SecurityGroupEgressArgs(
+                    protocol="-1",  # -1 means ALL protocols
+                    from_port=0,
+                    to_port=0,
+                    cidr_blocks=["0.0.0.0/0"],
+                )
+            ],
+            tags=default_tag,
+        )
+
+        bastion_ami = aws.ec2.get_ami(
+            filters=[
+                aws.ec2.GetAmiFilterArgs(
+                    name="name",
+                    values=["amzn2-ami-hvm-*"],
+                ),
+            ],
+            owners=["137112412989"],  # This owner ID is Amazon
+            most_recent=True,
+        )
+
+        stack_name = pulumi.get_stack()
+
+        keypair = aws.ec2.KeyPair(
+            "bastion-ssh-key",
+            key_name=f"bastion-{stack_name}-ssh-key",
+            public_key="ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEA14swrmHESs/dE8+nhj8twLPpbq//FHAVw7fXEQIvflJ+03yb6ostNLcXNroenoKzfRAwqVx0BRgYB0RKoBP5yIL7K5UkzxnWHEpIj3oC+kIrR1kn8LlsGa1aP05BYmZvsBXFzBjiBie3vhOTGOiEVEzqxdJLZExa6HIeoarKukAi94uQNDFZbnBNqdjRdgIPUdDHYlL5abSKC+yTS2+L96y6Dzt47i+rkn2rZzld1nYgRNLfAtvooIPZ60IfCnhRGPGuPA+nyYLtgRYVKCfC5yZ6QFUp0WH9NjE7PJMY2UX6w465bgbWlVtu8Ue687oWNyJf3r39S73gmpHOUHMkuw== opyate@gmail.com",
```

Comment:
Oh, just realised this might be a problem for bastion usage. I couldn't find a way to incorporate more public keys in here. Maybe one for a near-future task?
You can always just temporarily plonk your own public key in here if you need to use the bastion.

Reply:
Or perhaps we set up a shared public/private key combo, and put it in our respective .ssh folders.

Reply:
would be good to work out how we use AWS creds to access this - sharing keys is a pain in the long run & is a pain if someone needs to have their access revoked.

Reply:
https://github.com/climatepolicyradar/navigator/issues/484